kuma/static/js/tooltip.js

```diff
@@ -63,42 +57,6 @@
             });
         }
 
-        /**
```

Comment:
Nice to be able to remove so much code. I was surprised, though. Given the name of the file, I assumed that it was intended as a fully-generic tooltip implementation, so I was suprised to realize that we're only using it for one specific pre-positioned tooltip. I'd suggest adding a comment at the top of the file with that context and a pointer to the scss file for the email tooltip.

Reply:
I was thinking that we can definitely turn it into one, but for now, it makes more sense to couple it to the payments bit.